torch/distributed/_spmd/api.py

```diff
@@ -271,6 +271,9 @@ def _fused_adam_decomp(
     aten._foreach_mul_.Scalar: partial(
         _foreach_binop_scalar_decomp, aten._foreach_mul.Scalar
     ),
```

Comment:
Are we actively developing SPMD? 😮

Reply:
Unsure but I had a distributed test failure when I updated foreach Adam to use inplace div and this should fix.

If this is no longer needed, we should remove the tests 😛 and the code 😛 which I'm thinking would be a bigger more involved change than this.

Reply:
Let me disable the tests.